src/entity/posts/utils.ts

```diff
@@ -74,17 +74,22 @@ export const mergeKeywords = async (
   keywords?: string[],
 ): Promise<{ mergedKeywords: string[]; allowedKeywords: string[] }> => {
   if (keywords?.length) {
+    const cleanedKeywords = uniqueifyArray(
+      keywords
+        .map((keyword) => keyword.replace(/[^a-zA-Z0-9-_#.]/g, ''))
```

Comment:
It would be cool to extract this regex as const to avoid multiple instantiation and also name it something better to denote what it is trying to clean.